x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/context.ts

```diff
@@ -98,7 +98,21 @@ export function registerContextFunction({
             subscriber.complete();
           })
           .catch((error) => {
-            subscriber.error(error);
+            resources.logger.error('Error in context function');
+            resources.logger.error(error);
```

Comment:
Instead of propagating the error we log it, and return structured error as response